src/renault_api/cli/renault_vehicle.py

```diff
@@ -315,6 +316,28 @@ async def update_lock_status(
             update_status_table(status_table, key, value, unit)
 
 
+async def update_res_state(
+    vehicle: RenaultVehicle, status_table: Dict[str, Any]
+) -> None:
+    """Update status table from get_vehicle_res_state."""
+    try:
+        if not await vehicle.supports_endpoint("res-state"):  # pragma: no cover
+            return
+        response = await vehicle.get_res_state()
+    except QuotaLimitException as exc:  # pragma: no cover
+        raise click.ClickException(repr(exc)) from exc
+    except KamereonResponseException as exc:  # pragma: no cover
+        click.echo(f"res state: {exc.error_details}", err=True)
+    else:
+        items = [
+            ("RES details", response.details, None),
+            ("RES code", response.code, None),
```

Comment:
I don't think we need to show the code in the CLI. The details is probably sufficient.
```suggestion
            ("Engine state", response.details, None),
```

(that would need adjusting in the CLI test results)

Reply:
Done